vars/launchBuild.groovy

```diff
@@ -29,9 +29,7 @@ def call(ctx, jobName, jobId, buildName, toTag, dockerfile) {
       ],
       "spec": buildSpec
     ]
-    ctx.runAndWaitForBuild(ctx,
-      build.metadata.name,
-      { return ctx.openshift.apply(build) }
-    )
+    def buildJSON = groovy.json.JsonOutput.prettyPrint(groovy.json.JsonOutput.toJson(build))
```

Comment:
Why do we need to `prettyPrint`?

Reply:
Because it gets output in the Jenkins job log. It’s much easier to read.